mercure.rst

```diff
@@ -231,6 +231,19 @@ and to use URI Templates as patterns:
 
     Test if a URI Template match an URL using `the online debugger`_
 
+.. tip::
+
+    The native implementation of EventSource doesn't allow specifying headers.
+    For example, authorization using Bearer token. In order to achieve that, use `a polyfill`_
```

Comment:
Maybe can we add that using cookies is considered a best practice (because it’s more secure) when the client is a web browser?

Reply:
This phrase is already in Authorization section. I moved this tip in there.